packages/manager/apps/container/src/container/nav-reshuffle/onboarding/walkMe/index.jsx

```diff
@@ -0,0 +1,234 @@
+  const [arrowPlacement, setArrowPlacement] = useState();
+  const [popperInstance, setPopperInstance] = useState();
+
+  const {
+    closeOnboarding,
+    openAccountSidebar,
+    closeAccountSidebar,
+    openNavigationSidebar,
+    closeNavigationSidebar,
+  } = useProductNavReshuffle();
+
+  const [currentStepIndex, setCurrentStepIndex] = useState(0);
+  const [isPopoverVisible, setIsPopoverVisible] = useState(false);
+  const [isMobile, setIsMobile] = useState(
+    window.innerWidth < MOBILE_WIDTH_RESOLUTION,
+  );
+
+  const steps = [
+    {
+      selector: '#header-user-menu-button',
+      placement: 'bottom-start',
+      title: t('onboarding_walkme_popover_step1_title'),
+      content: t('onboarding_walkme_popover_step1_content'),
+    },
+    {
+      selector: '#user-account-menu-profile',
+      placement: 'left-start',
+      mobilePlacement: 'bottom-start',
+      title: t('onboarding_walkme_popover_step2_title'),
+      content: t('onboarding_walkme_popover_step2_content'),
+      onBeforeEnter: () => {
+        openAccountSidebar();
+        const animationPromise = new Promise((resolve) => {
+          setTimeout(() => {
+            resolve();
+          }, 150);
+        });
+        return animationPromise;
+      },
+    },
+    {
+      selector: '#user-account-menu-payment-method',
+      placement: 'left-start',
+      mobilePlacement: 'bottom-start',
+      title: t('onboarding_walkme_popover_step3_title'),
+      content: t('onboarding_walkme_popover_step3_content'),
+    },
+    {
+      selector: '#sidebar-link-services',
+      placement: 'right-start',
+      mobilePlacement: 'bottom-end',
+      title: t('onboarding_walkme_popover_step4_title'),
+      content: t('onboarding_walkme_popover_step4_content'),
+      onBeforeEnter: () => {
+        closeAccountSidebar();
+        if (isMobile) {
+          openNavigationSidebar();
+        }
+      },
+    },
+    {
+      selector: '#sidebar-link-billing',
+      placement: 'right-start',
+      mobilePlacement: 'bottom-end',
+      title: t('onboarding_walkme_popover_step5_title'),
+      content: t('onboarding_walkme_popover_step5_content'),
+    },
+  ];
+
+  const onHideBtnClick = (onboardingStatus) => {
+    closeOnboarding(onboardingStatus);
+    closeAccountSidebar();
+    if (isMobile) {
+      closeNavigationSidebar();
+    }
+  };
+
+  const onNextBtnClick = () => {
+    if (currentStepIndex + 1 < steps.length) {
+      setIsPopoverVisible(false);
+      setCurrentStepIndex(currentStepIndex + 1);
+    } else {
+      onHideBtnClick(ONBOARDING_STATUS_ENUM.DONE);
+    }
+  };
+
+  const calculateTargetBound = useCallback(() => {
+    const currentStep = steps[currentStepIndex];
+    const targetElement = document.querySelector(currentStep.selector);
+    const targetPos = targetElement.getBoundingClientRect();
+    const el = stepElement.current;
+
+    const positionOffset = ELEMENT_OFFSET / 2;
+
+    el.style.top = `${targetPos.top - positionOffset}px`;
```

Comment:
Out of curiosity there is no library for React that already handles such onboarding ?

Reply:
There is intro-js that have a react wrapper but it requires a license for commercial apps.
I didn't find other alternatives